dali/operators/image/paste/multipaste.cu

```diff
@@ -25,61 +25,70 @@ inline void to_vec(vec<n, T> &out, Source &&src) {
     out[i] = src[i];
 }
 
-void MultiPasteGPU::InitSamples(const TensorListShape<> &out_shape) {
+void MultiPasteGPU::InitSamples(const Workspace &ws, const TensorListShape<> &out_shape) {
   assert(spatial_ndim_ == 2);
+  assert(out_shape.sample_dim() == 3);
```

Comment:
`DALI_ENFORCE`?

Reply:
I went for an assert because we validate the input shape to have ndim == 3 and set the output shape ourseleves requiring the same dimensionality, so it should be met. Or is there a gap?

Reply:
Makes sense, thanks